zingoconfig/src/lib.rs

```diff
@@ -74,6 +79,8 @@ impl ZingoConfig {
             monitor_mempool: false,
             reorg_buffer_offset: REORG_BUFFER_OFFSET,
             zingo_wallet_dir: dir,
```

Comment:
Maybe it's a good idea to change `zingo_wallet_dir` to `wallet_dir`, I don't know if this can be so much trouble...

Reply:
Done.